### DISCUSSION TITLE
about open cv some questions

### DISCUSSION THREAD

**Discussion by Field-of-Dream** [community/community#109169] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can the Open CV library be able to read the camera and identify specific objects?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Field-of-Dream ! We're happy you're here. The Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. 

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
